packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -290,7 +290,12 @@ export const resolveBindingNode = (
       extensions: ['.js', '.jsx', '.ts', '.tsx'],
     });
     const moduleCode = fs.readFileSync(modulePath, 'utf-8');
```

Comment:
can we move this in the cache too please

Reply:
🤔 What if module code changes between the builds? When we go for file caching?

Reply:
we could use last modified date maybe? 🤔 

only a problem for file caching at least

Reply:
🤔 hmm. Let me try something.